test/test_uri_spec.py

```diff
@@ -181,6 +182,12 @@ def create_tests(test_path):
                 scenario_def = json.load(scenario_stream)
 
             for testcase in scenario_def['tests']:
+                if (testcase['uri'].startswith(SRV_SCHEME) and not
```

Comment:
Please move this skipping logic to the `run_scenario` method above and use self.skip(...) instead of print.